scripts/devnet_message.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
```

Comment:
I wonder if we could generalize this for testnet too? Not necessary in this PR though.